Sources/WebDriver/Session.swift

```diff
@@ -292,6 +301,20 @@ public class Session {
         try webDriver.send(Requests.SessionKeys(session: id, value: value))
     }
 
+    /// Change focus to another window
+    /// - Parameter name: The window to change focus to
+    public func window(name: String) throws {
```

Comment:
Does `focus(window:)` make more sense for this operation?

Reply:
I like these names better I'll push up a change